tests/appsec/iast/test_iast_path_traversal.py

```diff
@@ -18,7 +18,7 @@ class TestIastPathTraversal:
 
     EXPECTATIONS = {
         "java": {"LOCATION": "com.datadoghq.system_tests.springboot.iast.utils.PathExamples"},
-        "nodejs": {"LOCATION": "/usr/app/iast.js"},
+        "nodejs": {"LOCATION": "iast.js"},
```

Comment:
will it fail in nightly builds? Nightly builds test the latest snapshot and the latest release.